mods/ra2/rules/world.yaml

```diff
@@ -367,12 +365,36 @@ World:
 			5: FiveMinutesRemaining
 			10: TenMinutesRemaining
 			20: TwentyMinutesRemaining
+	PathFinderOverlay:
+	HierarchicalPathFinderOverlay:
+	ColorPickerManager:
+		PreviewActor: amcv.colorpicker
+		PresetHues: 0, 0.125, 0.185, 0.4, 0.54, 0.66, 0.79, 0.875, 0, 0.14, 0.23, 0.43, 0.54, 0.625, 0.77, 0.85 # Copied from RA.
+		PresetSaturations: 0.9, 0.9, 0.9, 0.9, 0.9, 0.9, 0.9, 0.9, 0.4, 0.5, 0.4, 0.5, 0.4, 0.5, 0.4, 0.5 # Copied from RA.
```

Comment:
The comments here can be removed I guess.

Reply:
As you wish. I figured it may be good for traceability so someone doesn't wonder how the hell these numbers came to be in a year or two.